rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/values.kt

```diff
@@ -233,13 +233,17 @@ data class UnlimitedStringValue(var value: String) : AbstractStringValue {
  * Cp0280   EBCDIC ITALIAN
  * See: https://www.ibm.com/support/knowledgecenter/SSLTBW_2.1.0/com.ibm.zos.v2r1.idad400/ccsids.htm
  */
-fun sortA(value: Value, charset: Charset) {
+fun sortA(value: Value, type: Type, charset: Charset) {
```

Comment:
If you assume that the only type accepted is `ArrayType`, why have you defined the new parameter type as `Type`  and not `ArrayType?`.
If you allow to pass any instance of `Type`,  the first time that I use `sortA` and I pass for example `StringType,` we will have a `ClassCastException`

Reply:
Fixed